ROADMAP.rst

```diff
@@ -0,0 +1,96 @@
+* Furthermore, not all the features implemented in the logic layer need be exposed by Mu. Discussion of MegaMu (a more featureful version of Mu) has taken place and providing a selection of features for UI designers to pick'n'mix depending on the needs of their users would make Mu a compelling solution for editor designers.
+* How the logic and interface layers interact should be clearly documented.
+
+Generic Editor
+==============
+
+* The RPi Foundation have asked that we support other Python contexts. For example, having the ability to create / write PyGameZero, GPIOZero and NetworkZero based projects with the UI automatically updating as the context changes. I have an old branch that provides a model for this in the logic layer that reaches into the interface layer. It's an exploratory spike that I believe could bear fruit.
+* To help in the user's graduation to a "real" editor we should ensure generic editor behaviours work as per the conventions: shortcut keys are the same, editing activities (e.g. selecting and acting upon blocks of code) should work as expected and helpful features (like code completion and call tips) should work in "the usual way".
+
+Multi Device Support
+====================
+
+* Adafruit, Calliope and others have expressed an interest in Mu supporting their devices. It should, and do so automagically. It should be possible to use USB's manufacturer and device ID's to work out what's plugged in and adapt accordingly.
+* It should be obvious from the source code how to add newly supported devices. This may be related to the "Python contexts" work mentioned in the Generic Editor section above. Such work should at least encompass adding flashing a device with the appropriate version of MicroPython, copying / flashing scripts to the device and connecting to an on-board file-system and connecting to the REPL.
+
+New Features
+============
+
+* Carlos has a spike proof of concept for Blockly integration (i.e. visual programming a la Scratch - move blocks around to create a program and it emits Python). Should we implement this (and I believe many teachers would love to see this) we *must* address the problem of migrating beginners from a visual style of programming to text based programming. This could be addressed by some of the UI related points from above.
+* The REPL, while an amzingly powerful feature, can appear a little unfriendly to beginners. Jupyter notebooks are a replacement for the REPL but also allow additional annotations and images to interweave with the code to be evaluated. That they can be shared and are fully editable also means they're a great way to share a "movement of thought" in words, code and pictures. I'd like to explore how these could be used to deliver lesson plans, exercises and "follow-along" resources for teachers and learners.
+
+Documentation
```

Comment:
once additional hardware is supported in mu, adafruit can assist with some "Getting Started with Mu" guides as well as some videos, let me know what would be helpful once that's in place.